Examples/Python/src/EventData.cpp

```diff
@@ -36,9 +36,20 @@ void addEventData(Context& ctx) {
                                     [](py::object /* self */) {
                                       return Acts::ParticleHypothesis::pion();
                                     })
-      .def_property_readonly_static("electron", [](py::object /* self */) {
-        return Acts::ParticleHypothesis::electron();
-      });
+      .def_property_readonly_static(
```

Comment:
Does this not work:

```cpp
.def_property_readonly_static("electron", &Acts::ParticleHypothesis::electron)
```

?

Reply:
I tried but this results in a segfault. not sure why tho. reverted that changes